.github/workflows/tekton_task_tests.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           echo ${{ steps.changed-dirs.outputs.all_changed_files }}
           trustedArtifactsBasedTasks=()
+          pvcBasedTasks=()
```

Comment:
The fact that a lot of tasks no longer support the pvc mode should probably be reflected in the contributing guide.
https://github.com/konflux-ci/release-service-catalog/blob/development/CONTRIBUTING.md#running-tekton-task-tests-manually

> Optionally enable Trusted Artifact mode by setting the environment variable:
> 
> export USE_TRUSTED_ARTIFACTS=true
> 
> Note: if a task has not been converted, and you have enable Trusted Artifacts, you will see errors.

Are there still tasks that support both now? Maybe? But most will probably support one or the other, right? So this snippet should probably say something about that.

Reply:
totally right! I did not even think about that.

My desire would to convert all tasks and pipelines to Trusted Artifacts.

As it stands in this PR, the `pvcBasedTasks` check is primarily for `tasks/internal` and `tasks/collectors` that will may use PVC workspaces.

Again, we'll keep this open to remind me to update docs should this go forward.